import "IpcMessage.proto";

package METRO.CRO.MESSAGES;

message RequestLogin {
	extend IpcMessage {
		optional RequestLogin message = 105;
	}

	required string username = 1;
	required string password = 2;
}